scripts/e2e-tests.sh

```diff
@@ -261,30 +259,24 @@ function setup_test_cluster() {
   set -o errexit
   set -o pipefail
 
-  # Set the required variables if necessary.
-  if [[ -z ${K8S_USER_OVERRIDE} ]]; then
-    export K8S_USER_OVERRIDE=$(gcloud config get-value core/account)
-  fi
+  k8s_user=$(gcloud config get-value core/account)
+
+  USING_EXISTING_CLUSTER=0
+
+  k8s_cluster=$(kubectl config current-context)
+  acquire_cluster_admin_role ${k8s_user} ${E2E_CLUSTER_NAME} ${E2E_CLUSTER_REGION} ${E2E_CLUSTER_ZONE}
```

Comment:
I'm not 100% sure this is necessary. Actually, I'm even concerned that it will fail (if not the first time, the second time, because the rolebinding already exists) for a cluster already configured. Probably the best way here is to detect somehow that the binding already exists; if so, then much probably you're using an existing cluster (then you set the var). Otherwise, call `acquire_cluster_admin_role`.